draft-ietf-gnap-core-protocol.md

```diff
@@ -1485,18 +1501,22 @@ The following sections detail requests for interaction
 modes. Additional interaction modes are defined in 
 [a registry TBD](#IANA).
 
+#### Examples
```

Comment:
Just to help it render for now -- otherwise this section gets an auto-generated "#examples" anchor that conflicts with another section, and this keeps the draft from building.

```suggestion
#### Example Interactions
```